src/sys/socket/addr.rs

```diff
@@ -1124,6 +1144,66 @@ mod datalink {
     }
 }
 
+#[cfg(target_os = "linux")]
+pub mod vsock {
+    use ::sys::socket::addr::AddressFamily;
+    use libc::{sa_family_t, sockaddr_vm};
+    use std::{fmt, mem};
+    use std::hash::{Hash, Hasher};
+
+    #[derive(Copy, Clone)]
+    pub struct VsockAddr(pub sockaddr_vm);
+
+    // , PartialEq, Eq, Debug, Hash
```

Comment:
What exactly does this comment mean, and why the leading comma?

Reply:
Oh, sorry about that! I think was a line copied by struct AlgAddr that I used as a template for VsockAddr. I'll remove this line.